### PR TITLE
add a testClass for automation purpose

### PR DESCRIPTION
**Pull request by orangeflame**

### CODE REVIEW

File: src/containers/SaveBarHandler/SaveBarButton/index.jsx [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 import React from 'react';
+import classnames from 'classnames';
 import Button from 'components/Button';
 import PropTypes from 'prop-types';
+import testClass from 'domain/testClass';
 
-const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) =>
-  <Button autoWidth
+const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) => {
+  const className = classnames(styles.SaveBar_button, testClass(`${label}-btn`));
```
Comment (rulych) [inline src/containers/SaveBarHandler/SaveBarButton/index.jsx]: `label` here is the text inside the button. That could be `Do Something`, with a space. The CSS class will be broken.

File: src/containers/SaveBarHandler/SaveBarButton/index.jsx [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+import classnames from 'classnames';
 import Button from 'components/Button';
 import PropTypes from 'prop-types';
+import testClass from 'domain/testClass';
 
-const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) =>
-  <Button autoWidth
+const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) => {
+  const className = classnames(styles.SaveBar_button,
+    testClass(`${label.replace(/\s+/g, '')}-btn`));
```
Comment (rulych) [inline src/containers/SaveBarHandler/SaveBarButton/index.jsx]: Let's try to make it nicer than this. `test-DoSomething-btn` or `test-Dosomething-btn`, etc. would look horrible.

We can replace spaces with dashes `-`, and make sure everything is lowercase with something like `label.replace(/\s+/g, '-').toLowerCase()`.

Comment (rulych) [inline src/containers/SaveBarHandler/SaveBarButton/index.jsx]: Another thing is that we want these classes to be as unique as possible. How about `test-toolbar-button-<label>` instead?

Comment (orangeflame) [inline src/containers/SaveBarHandler/SaveBarButton/index.jsx]: Sure

**Pull request merged**